install/zoraxy-install.sh

```diff
@@ -20,20 +20,11 @@ $STD apt-get install -y mc
 $STD apt-get install -y git
```

Comment:
do you need `git`?

Reply:
I thought it might be needed by the `*.func`-magic in the shadows.
Removed.